### PR TITLE
Change in Colors and width of scroll bar

### PR DESCRIPTION
**Pull request by Tapo41**
color and the width of the scroll bar is not correct ,i'm correcting that

### CODE REVIEW

**Review by arpitghura — COMMENTED** [review]
<h1>It's great having you contribute to this project</h1> Welcome to the community :nerd_face: Thanks for giving your time to contribute to <b>Typing Test</b> Project.

**Comment by arpitghura** [comment]
Hey @Tapo41, Add screenshots of the changes reflecting on the website.

**Comment by Tapo41** [comment]
Screenshot_20230211_173316
Before 
Screenshot_20230211_174937

After changing

**Comment by Tapo41** [comment]
@arpitghura First of all sorry for the late reply .
Please check the screenshot after zooming into the desktop size. Then only you can able to notice the actual changes made.
Before the scroll bar was not properly visible due to color overlapping issue. Now it is resolved.

File: assets/css/index.css [hunk 1]
```diff
@@ -21,17 +21,18 @@ main{
 /* ScrollBar */
 /* width */
 ::-webkit-scrollbar {
-  width: 10px;
+  width: 15px;
```
Comment (arpitghura) [inline assets/css/index.css:24]: Do not increase the width of the scrollbar

**Review by arpitghura — CHANGES_REQUESTED** [review]

File: assets/css/index.css (hunk 1 above)
Comment (Tapo41) [reply assets/css/index.css]: @arpitghura ok updated

**Pull request merged**